dist/templates/decanter/components/local-footer/local-footer.twig

```diff
@@ -101,7 +103,7 @@
     responsive behaviour so they are not optional. Empty cells will still
     render and create whitespace.
   #}
-  <section class="su-local-footer__columns">
+  <div class="su-local-footer__columns">
```

Comment:
@jdwjdwjdw Why did you change from a `section` to a `div`? I believe it is there for accessibility reasons. It groups similar content. Let's check with @cjwest before switching it out.

Reply:
Yes I just tried it on Voiceover. We don't want to remove those sections. I'll show you why in the meeting later.